README describes a non-functional install step

**Describe the bug**

README section that guides the user to install dependencies into local Maven cache no longer works. A replacement command is needed, potentially `./gradlew check` or something similar.

**To Reproduce**

Go to [README install section](https://github.com/spring-projects/spring-authorization-server#install-all-spring--jars-into-your-local-maven-cache), and run the `install` Gradle task as described. Gradle wrapper then throws: 

```text
FAILURE: Build failed with an exception.

* What went wrong:
Task 'install' not found in root project 'spring-authorization-server'. Some candidates are: 'npmInstall'.
```

**Expected behavior**

Maven dependencies of the entire project are preloaded into local Maven cache, and task succeeds.

--- a/README.adoc
+++ b/README.adoc
@@ -66,7 +66,7 @@ git clone git@github.com:spring-projects/spring-authorization-server.git
 === Install all spring-\* jars into your local Maven cache
 [indent=0]
 ----
-./gradlew install
+./gradlew publishToMavenLocal
 ----
 
 === Compile and test; build all jars, distribution zips, and docs

